cmd/playground/playground_window.go

```diff
@@ -0,0 +1,38 @@
+package playground
```

Comment:
change the file name from `playground_window.go` to be just `window.go` since we already know that this is inside a `playground`